ongebo/kabedo
# Design for profile update user interface and experience

### Goal to Achieve
Addition of design to enable users to update their profiles
#### Changes to Make
- Add HTML, CSS, and JS for profile update user interface and experience
- Change JS indentation from two to four spaces
- Comment HTML sections in `homepage.html` for easy readability
- Limit input styles in styles.css to form elements
#### Manual Testing
- Fetch and check out this branch (`origin/profile-update-164847546`)
- Open `ui/html/user-profile.html` in a browser
- Click the edit links to see how users will update their profiles
#### Screenshots
**Update Links on User Profile Page**
![image](https://user-images.githubusercontent.com/34346070/63284104-89ea0900-c2bb-11e9-9f1a-be1aa9ca9dd8.png)

**Bio Update Modal**
![image](https://user-images.githubusercontent.com/34346070/63284216-bbfb6b00-c2bb-11e9-9819-d84180037cfa.png)

**Hobbies Update Modal**
![image](https://user-images.githubusercontent.com/34346070/63284225-c158b580-c2bb-11e9-8592-8461630007af.png)

#### Relevant PT Stories
[#164847546](https://www.pivotaltracker.com/story/show/164847546)
